knowledge-base/html5-viewer-does-not-load-with-jquery-slim.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Kendo could not be loaded automatically error.
+description: The HTML5 Report Viewers do not load when using the slim build of jQuery.
+type: troubleshooting
+page_title: Kendo could not be loaded automatically error is thrown in the browser console.
+slug: html5-viewer-does-not-load-with-jquery-slim
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Application Type</td>
+			<td>Web Application</td>
+		</tr>
+		<tr>
+			<td>Report Viewer</td>
+			<td>HTML5 Report Viewers</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+The error is displayed in the browser console when the report viewer tries to load and a message in the report viewer that the Reporting REST Service cannot be accessed.
```

Comment:
Consider splitting into 2 sentences, for example:
The error is displayed in the browser console when the report viewer loads. 'The Reporting REST Service cannot be accessed' message appears in the report viewer.